src/MainWindow.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>988</width>
-    <height>632</height>
+    <height>759</height>
```

Comment:
is that actually expanding our UI?

Reply:
It is always auto generated, when you open the MainWindow.ui in Designer and set to the value based on your screen. I think we should keep the original values. (I always change them back)

Reply:
Hi deXol!

Thanks for fixing it for me! If you prefer let me know next time and I can take care of fixing my bugs. :-)